src/components/Navbar.tsx

```diff
@@ -51,8 +51,8 @@ function Group({ links }: { links: { href: string, text: string }[] }) {
         }
       />
       {links.map(({ href, text }, i) => i == 0
-        ? <NavLink class="z-10 pl-14 lg:pl-0 p-4" href={links[0].href}>{links[0].text}</NavLink>
-        : <NavLink
+        ? <NavLink key={i} class="z-10 pl-14 lg:pl-0 p-4" href={links[0].href}>{links[0].text}</NavLink>
+        : <NavLink key={i}
```

Comment:
Right - i forgot about this!

It should be fine since `links` will probably never change, but i think it would be more correct to use `href` as key

Also, the `links[0].{href,text}` was a copy paste-error. They should just be `href` and `text`!

Reply:
I've fixed these issues now and rebased to the same commit so we don't have to create a new PR.